# gains of the contact point estimator
gamma_r: 7.0
kappa_r: 0.0
beta_r: 0.2

# initial contact point (in FT sensor frame)
initial_r: [0.2, 0.2, 0.1]

# update frequency of contact point estimator [Hz]
cpe_update_frequency: 650.0